### PR TITLE
refactor: high level mod structure

### PR DESCRIPTION
**Pull request by nathanosdev**
This PR refactors the high level mod structure of the response verification package. There's no meaningful code changes.

### CODE REVIEW

File: packages/ic-response-verification/src/cbor.rs [hunk 1]
```diff
@@ -1,5 +0,0 @@
-pub mod certificate;
```
Comment (nathanosdev) [inline packages/ic-response-verification/src/cbor.rs:1]: moved to `cbor/mod.rs`

File: packages/ic-response-verification/src/cbor/mod.rs [hunk 2]
```diff
@@ -0,0 +1,5 @@
+pub mod certificate;
```
Comment (nathanosdev) [inline packages/ic-response-verification/src/cbor/mod.rs:1]: moved from `cbor.rs`

File: packages/ic-response-verification/src/hash.rs [hunk 3]
```diff
@@ -1,15 +0,0 @@
-//! Utilities for calculating
```
Comment (nathanosdev) [inline packages/ic-response-verification/src/hash.rs:1]: moved to `hash/mod.rs`

File: packages/ic-response-verification/src/hash/mod.rs [hunk 4]
```diff
@@ -1,7 +1,20 @@
+//! Utilities for calculating
```
Comment (nathanosdev) [inline packages/ic-response-verification/src/hash/mod.rs:1]: Moved from `hash.rs`

**Review by keplervital — APPROVED** [review]

**Pull request merged**